src/shared-components/avatar/AvatarWithDetails/AvatarWithDetails.stories.tsx

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2025 New Vector Ltd.
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only OR GPL-3.0-only OR LicenseRef-Element-Commercial
+ * Please see LICENSE files in the repository root for full details.
+ */
+
+import React from "react";
+
+import { AvatarWithDetails } from "./AvatarWithDetails";
+import type { Meta, StoryFn } from "@storybook/react-vite";
+
+export default {
+    title: "Avatar/AvatarWithDetails",
+    component: AvatarWithDetails,
+    tags: ["autodocs"],
+    args: {
+        avatar: <div style={{ width: 40, height: 40, backgroundColor: "#888", borderRadius: "50%" }} />,
+        details: "Details about the avatar go here",
+        roomName: "Room Name",
```

Comment:
If I was being critical here I'd just call this title or name, since the rest of the component doesn't refer to rooms (which makes this more generic)